src/DotNet.Status.Web/Models/AnnotationEntry.cs

```diff
@@ -0,0 +1,35 @@
+namespace DotNet.Status.Web.Models
+{
+    public class AnnotationEntry
+    {
+        /// <summary>
+        /// The original annotation sent from Grafana.
+        /// </summary>
+        public Annotation Annotation { get; set; }
+        /// <summary>
+        /// Time since UNIX Epoch in milliseconds. (required)
+        /// </summary>
+        public long Time { get; set; }
+        public long? TimeEnd { get; set; }
+        public bool? IsRange { get; set; } = false;
+        /// <summary>
+        /// The title for the annotation tooltip. (required)
+        /// </summary>
+        public string Title { get; set; }
+        /// <summary>
+        /// Tags for the annotation. (optional)
+        /// </summary>
+        public string[]? Tags { get; set; }
+        /// <summary>
+        /// Text for the annotation. (optional)
+        /// </summary>
+        public string? Text { get; set; }
```

Comment:
We don't use nullability annotationsin this project. I'm actually sort of suprized this compiled at all.

Reply:
Oops, will remove. 

It issues a warning.

Reply:
I could enable nullability for this section, as an incremental step towards updating the whole project.

Reply:
Please don't. The only thing worse than no nullability is partial nullability.